.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+- '4.1'
+- '4.0'
+- '0.12'
+- '0.11'
+- '0.10'
+notifications:
+  slack:
+    secure: lguGyEEsQCqB1gbJPuKeWPzDLXivGgM2bflu5AJgxBHOl7N19MwiUBg46mP2qeImuSaPcGmnsAn6pjNQHgPFAuhH23T7vSyAbL7/cUIGYOBZ7teTuWKzlESirxcjNMQ0SLHRGXb959XMXspyVkO0OGP7yFz0MNnVyihTQaHQSR82ZDXuuLEFtnhh5DwVN3XjXiis7wJXXmasQN+XWyVBcuE/AUY1AV9gq6igL7IfoV86DzPHj67Z9uLAJMEANj0Z7sNVU1dkUxqtRGYBGBEX0Mxjq2hydJjekrUCOdNPIgFYS5GrNkgtK1d0TC5oobn8YHPmQ51Pcd70UPenP8K6mvCEm2xR4uPvBaSIIozn4SJDSYkGzqZQXNZDoXo2edk1MlE3NHsbdui9bZo7wJ+5KOLScKKwFMS6c4SGOxLY1OTChnvQgzjbDVEeCxEamlZwfizsy6Yrehf2W7cV2hyJpVQPDhlNzFgq8eEk1RJRjqRWcKJshQ4iFGeV38sZ+VPeeV88CdXnAQx4579enP1TMq56Y+QnaPiqkg3TZ1JjKkO4LdhMzGEUFK/JIrJdakcQZDwLZiFuCwPdKAHPUaGsXCXMQTO6RHsCfo8pPFQz0Va8sUWmvdr8rfFjCCaW/05Hxf1NjiIPI2lqV5RouV7OPbWz3hNSvY7tUcUULvtKY5o=
```

Comment:
Cool!

Only one way to find out.

-- 
Bob Van Landuyt
+32 473 43 05 97
On 5 Nov 2015, at 12:23, Piet Jaspers wrote:

>> @@ -0,0 +1,10 @@
>> +language: node_js
>> +node_js:
>> +- '4.1'
>> +- '4.0'
>> +- '0.12'
>> +- '0.11'
>> +- '0.10'
>> +notifications:
>> +  slack:
>> +    secure: lguGyEEsQCqB1gbJPuKeWPzDLXivGgM2bflu5AJgxBHOl7N19MwiUBg46mP2qeImuSaPcGmnsAn6pjNQHgPFAuhH23T7vSyAbL7/cUIGYOBZ7teTuWKzlESirxcjNMQ0SLHRGXb959XMXspyVkO0OGP7yFz0MNnVyihTQaHQSR82ZDXuuLEFtnhh5DwVN3XjXiis7wJXXmasQN+XWyVBcuE/AUY1AV9gq6igL7IfoV86DzPHj67Z9uLAJMEANj0Z7sNVU1dkUxqtRGYBGBEX0Mxjq2hydJjekrUCOdNPIgFYS5GrNkgtK1d0TC5oobn8YHPmQ51Pcd70UPenP8K6mvCEm2xR4uPvBaSIIozn4SJDSYkGzqZQXNZDoXo2edk1MlE3NHsbdui9bZo7wJ+5KOLScKKwFMS6c4SGOxLY1OTChnvQgzjbDVEeCxEamlZwfizsy6Yrehf2W7cV2hyJpVQPDhlNzFgq8eEk1RJRjqRWcKJshQ4iFGeV38sZ+VPeeV88CdXnAQx4579enP1TMq56Y+QnaPiqkg3TZ1JjKkO4LdhMzGEUFK/JIrJdakcQZDwLZiFuCwPdKAHPUaGsXCXMQTO6RHsCfo8pPFQz0Va8sUWmvdr8rfFjCCaW/05Hxf1NjiIPI2lqV5RouV7OPbWz3hNSvY7tUcUULvtKY5o=
>
> [More info](http://docs.travis-ci.com/user/encryption-keys/)
>
> It's basically encrypted on a per repo basis, so I'm wondering if this key will still work once this gets pulled into the repo. (I'd say no)
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/10to1/hubot-the-visual-aid/pull/5/files#r44001194